tomviz/ModuleOrthogonalSlice.cxx

```diff
@@ -140,6 +140,9 @@ void ModuleOrthogonalSlice::addToPanel(QWidget* panel)
     << "Slice";
   proxiesWidget->addProxy(this->Representation, "Slice", reprProperties, true);
   proxiesWidget->updateLayout();
+
+  this->connect(proxiesWidget, SIGNAL(changeFinished(vtkSMProxy*)),
+                SLOT(requestRender()));
```

Comment:
You know you can just connect a SIGNAL to another SIGNAL to forward a signal. There is no need to make a SLOT just to connect and emit a signal, and it should work fine when dropping arguments (like vtkSMProxy)